app/models/git/exercise_reader.rb

```diff
@@ -179,4 +144,28 @@ def exercise_tree
     oid = entry[:oid]
     lookup(oid)
   end
+
```

Comment:
Can we delete this?

Reply:
Yeah - I left it there in case we wanted to revert, but we can just revert the PR in that situation.